main.py

```diff
@@ -146,11 +146,13 @@ def _restore_status(self):
             interface_status, lldp_status = state
             try:
                 switch = self.controller.switches[switch_id]
+                interface = switch.interfaces[interface_number]
+                self.update_instance_metadata(interface)
```

Comment:
The `update_instance_metadata` is implemented in:
 https://github.com/kytos/topology/blob/d9a87848c8a554516f0e64ad992b8d67140255eb/main.py#L761